pallets/operation/src/lib.rs

```diff
@@ -341,6 +349,25 @@ pub mod pallet {
             }
             Ok(().into())
         }
+
+        #[pallet::weight(T::OPWeightInfo::burn_for_ezc())]
+        pub fn burn_for_ezc(
+            origin: OriginFor<T>,
+            burned: BalanceOf<T>,
+            benifity: H160,
+        ) -> DispatchResultWithPostInfo {
+            let sender = ensure_signed(origin)?;
+
+            if T::Currency::can_slash(&sender, burned) {
+                let (burned, _) = T::Currency::slash(&sender, burned);
+                let balance = burned.peek();
+                T::BurnedTo::on_unbalanced(burned);
+                Self::deposit_event(Event::<T>::BurnForEZC(sender, balance, benifity));
```

Comment:
Is `balance` the same as `burned`?

Reply:
?

Reply:
`let balance = burned.peek();` is `balance` actually burned DPR amount?

Reply:
Yes，maybe using withdraw is more suitable